### PR TITLE
PWA: Cache Policy Added with Service Worker

### LINKED ISSUE #179: Site Performance : Serve the static content via the cache such as CDN or cache policy
There are more than 1MB of resources/assets which are being requested and loaded from the server, implement a cache policy  in the browser for static assets or serve it via CDN in order to improve the performance.

use lighthouse to determine the errors https://developers.google.com/web/tools/lighthouse/

### PR DESCRIPTION
**Pull request by Safnaj**
- Cache Policy Added for Static Contents
- Cache Policy Added for Medium API
- Disabled unwanted console logs
- PWA with Service Worker

Fixes #179

### CODE REVIEW

**Comment by Safnaj** [comment]
@Shehanka get a review from @sajeetharan before merge

File: src/app/components/blog/blog.component.ts
Comment (Shehanka) [inline src/app/components/blog/blog.component.ts:23]: remove these comments

**Review by Shehanka — CHANGES_REQUESTED** [review]
Remove that commented code. 

@sajeetharan we need a review u too

File: src/app/components/blog/blog.component.ts
Comment (Safnaj) [reply src/app/components/blog/blog.component.ts:23]: I thought to keep this for future need. Actually added by @Bawanthathilan

Comment (Shehanka) [reply src/app/components/blog/blog.component.ts:23]: can someone remove it

**Review by Shehanka — APPROVED** [review]

**Comment by sajeetharan** [comment]
@Shehanka  it's already merged right?

**Pull request merged**
